moduleroot/.travis.yml.erb

```diff
@@ -26,6 +25,8 @@ matrix:
 <% unless @configs['test_all_rvms'] -%>
     - rvm: 2.1.9
       env: PUPPET_VERSION=4.9
+    - rvm: 2.5.1
```

Comment:
Isn't this logically reversed? This should only be here if `test_all_rvms` is true.

Reply:
Hm, I don't get how that should work... If `test_all_rvms` is false only AIO configs (Puppet 4 with Ruby 2.1 and Puppet 5 with Ruby 2.4) should be tested, correct?

Reply:
OK, I think I got it now